src/FileSystem-Core/FileSystem.class.st

```diff
@@ -719,14 +719,21 @@ FileSystem >> readStreamOn: aResolvable [
 
 { #category : 'public' }
 FileSystem >> referenceTo: aResolvable [
-	"Answer a reference to the argument from the context of the receiver filesystem.
-		Example: Filesystem disk referenceTo: 'plonk.taz'"
-
+	"Answer a reference to the argument from the context of the receiver FileSystem.
```

Comment:
Hello 

In Pharo we have executable comment. an executable comment is defined between `""` and with a  `>>>`
Here is the code of the full method. Please not that the comment is correct. I mean "the receiver filesystem" is better than "the receiver FileSystem".

```
referenceTo: aResolvable
	"Answer a reference to the argument from the context of the receiver filesystem."
       "(FileSystem disk referenceTo: 'plonk.taz) class >>> FileReference"       


	^ FileReference
		fileSystem: self
		path: (self pathFromObject: aResolvable)
```

Reply:
Hi Ducasse,

I've updated the `referenceTo:` method as per your suggestions:

- Changed the descriptive comment to use "filesystem" instead of "FileSystem."
- Included the executable example within the comment block, using the `>>>` convention for expected output.

I am new to Pharo, so thank you for your guidance!

Please review the changes at your convenience. Let me know if there's anything else that needs adjustment.

Reply:
You are welcome. I did it on purpose this is important to help people get up to speed. 
So now you know that you can add executable comment to other methods of Pharo :).